FileStorage.Client/Program.cs

```diff
@@ -38,9 +38,40 @@ static void Main(string[] args)
             sw.Stop();
 
             Console.WriteLine(Environment.NewLine + "Transferred " + fi.Length + " bytes in " + sw.ElapsedMilliseconds);
+
+
+            string filenameRead = @"testPayloadRead.txt";
+            GetFile(filename, filenameRead);
+
             Console.ReadLine();
         }
 
+
+        private static void GetFile(string getFile, string saveFile)
+        {
+            HttpWebRequest webRequest = (HttpWebRequest) HttpWebRequest.Create("http://localhost:8081/Api/File/" + getFile);
+            webRequest.Method = "GET";
+
+            (webRequest as WebRequest).Headers.Add(HttpRequestHeader.Authorization,
```

Comment:
You don't really need this. It's used when identity server is enabled, I was just playing around and that's why it's there.